src/main/scala/rps/model/ResponseBody.scala

```diff
@@ -0,0 +1,6 @@
+package rps.model
+
+case class ResponseBody(result: Option[Result],
```

Comment:
minor, but `ResponseBody` refers explicitly to http, which isn't really a concern of your domain here. Try naming to something more apt to the domain model.

Reply:
good point 👍